packages/discovery-styles/scss/components/document-preview/_document-preview-pdf-viewer.scss

```
@@ -1,3 +1,61 @@
 .#{$prefix}--document-preview-pdf-viewer {
+  position: relative;
+}
+
+.#{$prefix}--document-preview-pdf-viewer--canvas {
+  transform-origin: left top 0px;
+}
+
+.#{$prefix}--document-preview-pdf-viewer--text {
   transform-origin: left top 0px;
+
+  //
+  // NOTE: import textLayer styles from ~pdfjs-dist/web/pdf_viewer.css
+  // @import "~pdfjs-dist/web/pdf_viewer" doesn't work for loading image
+  //
+  &.textLayer {
+    position: absolute;
+    text-align: initial;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    overflow: hidden;
+    opacity: 0.2;
+    line-height: 1;
+  }
+
+  &.textLayer span,
+  &.textLayer br {
+    color: transparent;
+    position: absolute;
+    white-space: pre;
+    cursor: text;
+    transform-origin: 0% 0%;
+  }
+
+  &.textLayer ::selection {
+    background: rgba(0, 0, 255, 1);
+  }
+
+  /* Avoids https://github.com/mozilla/pdf.js/issues/13840 in Chrome */
```

Please add more to this comment. In particular, a quick summary of the actual issue you're trying to work around. A reader shouldn't need to view that URL, unless they want to view more details. Something like:
```suggestion
  // Avoids ...some bug... in Chrome
  // @see https://github.com/mozilla/pdf.js/issues/13840
```